### ISSUE TITLE
Remove and ignore temporary files

### ISSUE THREAD

**Issue by petebachant** [pbnh/unh-aquaponics#1]
All the files ending in `~` don't need to be part of the repository. You should delete these, then check

```
git status
```

to make sure the correct files are being deleted.

Then create a `.gitignore` file in the project root directory with

```
touch .gitignore
```

and add this line to the file:

```
*~
```

Then commit all the changes you just made

```
git add .gitignore
git commit -am "Remove temporary files"
```

Then push to GitHub

```
git push
```

**Comment by petebachant** [comment]
Note that you can close this issue via commit message, if when you commit you write something like

```
git commit -am "Remove temporary files; resolves #1"
```

After you push, the issue will automatically be closed.

**Comment by petebachant** [comment]
Looks like this is done. Closing.
